web/src/server/api/routers/scores.ts

```diff
@@ -149,38 +159,49 @@ export const scoresRouter = createTRPCRouter({
         throw new Error("No trace with this id in this project.");
       }
 
-      const score = await ctx.prisma.score.create({
-        data: {
+      try {
+        const score = await ctx.prisma.score.create({
```

Comment:
use upsert to make sure that no score config can be applied twice via annotation workflow